src/main/java/com/example/music/backend/verification/domain/VerificationToken.java

```diff
@@ -0,0 +1,71 @@
+package com.example.music.backend.verification.domain;
+
+import com.example.music.backend.user.domain.User;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.util.Calendar;
+import java.util.Date;
+
+@Entity
+public class VerificationToken {
+
+    private static final int EXPIRATION = 60 * 24;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    private String token;
+
+    @OneToOne(targetEntity = User.class, fetch = FetchType.EAGER)
+    @JoinColumn(nullable = false, name = "user_id")
+    private User user;
+
+    private Date expiryDate;
+
+
+    public VerificationToken() {
+
+    }
+
+    public VerificationToken( String token, User user) {
+        this.token = token;
+        this.user = user;
+        expiryDate = calculateExpiryDate();
+    }
+    private Date calculateExpiryDate() {
```

Comment:
Entity is a bad place for calculation

Reply:
created a separate class VerificationTokenUtil